Src/Generic/Styling/TextStyle.swift

```diff
@@ -254,4 +254,376 @@ extension TextStyle {
         style.lineHeight = 20
         style.color = .secondaryText
     }
+    
+    static let headerLargeTitle = TextStyle.default.restyled { (style: inout TextStyle) in
+        style.font = HedvigFonts.circularStdBold!.withSize(30)
+        style.lineHeight = 32
+        style.color = .primaryText
+    }
+    
+    static let headlineLargeLargeRight = TextStyle.default.restyled { (style: inout TextStyle) in
+        style.font = HedvigFonts.soRayExtraBold!.withSize(24)
+        style.lineHeight = 24
+        style.alignment = .right
+        style.color = .primaryText
+    }
+    
+    static let headlineLargeLargeLeft = TextStyle.default.restyled { (style: inout TextStyle) in
+        style.font = HedvigFonts.soRayExtraBold!.withSize(24)
+        style.lineHeight = 24
+        style.color = .primaryText
+    }
+    
+    static let headlineLargeNegLargeNegLeft = TextStyle.default.restyled { (style: inout TextStyle) in
+        style.font = HedvigFonts.soRayExtraBold!.withSize(24)
+        style.lineHeight = 24
+        style.color = .switchPrimaryText
+    }
+    
+    static let headlineLargeLargCenter = TextStyle.default.restyled { (style: inout TextStyle) in
```

Comment:
I know this is wrong in zeplin too, but it should be `LargeLarge` not `LargeLarg` I think 😄